plugins/amdgpu/amdgpu_plugin.c

```diff
@@ -13,6 +13,7 @@
 #include <sys/mman.h>
 #include <sys/types.h>
 #include <stdint.h>
+#include <pthread.h>
```

Comment:
Don't you need to add -lpthread to the compiler command line in the Makefile somewhere?

Reply:
ACK. Forgot to git add Makefile and the change went into another commit in the other PR